packages/dx-react-grid/docs/reference/local-sorting.md

```diff
@@ -10,7 +10,10 @@ A plugin that performs local data sorting.
 
 ### Properties
 
-none
+Name | Type | Default | Description
+-----|------|---------|------------
+getColumnCompare | (columnName: string) => Function | | A function used to apply a custom sorting. Should return the sorting function like `(a, b) => number`. For more information see the [sorting guide](../guides/sorting.md#custom-sorting-algorithm).
```

Comment:
Please, specify interface completely. It should be understandable. For example: `(columnName: string) => (aValue: any, bValue: any) => number`